### PR TITLE
New holy weapon: Ratvarian Spear with throw-jaunting

### PR DESCRIPTION
**Pull request by silicons**
10 seocnd cooldown, if it hits something within 14 tiles of you you teleport to it, otherwise it teleports to you and you pick it up in both cases if you can.

### CODE REVIEW

File: code/game/objects/items/holy_weapons.dm [hunk 1]
```diff
@@ -419,6 +419,40 @@
 	sharpness = IS_SHARP
 	attack_verb = list("chopped", "sliced", "cut", "reaped")
 
+/obj/item/nullrod/ratvarian_spear
```
Comment (Poojawa) [inline code/game/objects/items/holy_weapons.dm]: you added this in the middle of the scythe code.

File: code/game/objects/items/holy_weapons.dm [hunk 2]
```diff
@@ -692,3 +691,38 @@
...
+		var/mob/living/L = throwingdatum.thrower
+		if(get_dist(src, L) < 14)
+			L.forceMove(get_turf(src))
+			L.visible_message("<span class='boldwarning'>[L] jaunts in with a shower of light!</span>")
+			L.put_in_hands(src)
+		else
+			visible_message("<span class='boldwarning'>[src] disappears with a crack!</span>")
+			forceMove(L.loc)
+			var/picked_up = L.put_in_hands(src)
+			to_chat(L, "<span class='boldwarning'>[src] materializes beneath you[picked_up? ", and you reflexively pick it up!" : ""]!</span>")
```
Comment (deathride58) [inline code/game/objects/items/holy_weapons.dm]: "The bronze spear materializes beneath you, and you reflexively pick it up!!"
```suggestion
			to_chat(L, "<span class='boldwarning'>[src] materializes beneath you[picked_up? ", and you reflexively pick it up" : ""]!</span>")
```

File: code/game/objects/items/holy_weapons.dm [hunk 3]
```diff
@@ -692,3 +691,38 @@
...
+		last_throw_special = world.time
+		to_chat(L, "<span class='boldwarning'>[src] clicks, its whirring growing softer. You probably won't be able to utilize its throw jaunt for [throw_special_cooldown/10] seconds!</span>")
```
Comment (deathride58) [inline code/game/objects/items/holy_weapons.dm]: time for micro-optimizations, grmbl grmbl
```suggestion
		to_chat(L, "<span class='boldwarning'>[src] clicks, its whirring growing softer. You probably won't be able to utilize its throw jaunt for [throw_special_cooldown*0.1] seconds!</span>")
```

**Review by deathride58 — CHANGES_REQUESTED** [review]

**Comment by Ghommie** [comment]
This reminds me of the old Sky Bulge (yes, it was its actual name) pogo-spear.

**Pull request closed without merging**